### PR TITLE
fix component

### PR DESCRIPTION
**Pull request by matthewmueller**
Fixes the example on the home page and strips initial indentation so markdown can properly render.

### CODE REVIEW

**Comment by matthewmueller** [comment]
fixes: https://github.com/acdlite/react-remarkable/issues/4

fyi, if you want to just use the fork `npm install matthewmueller-react-remarkable`. repo is here: http://github.com/matthewmueller/react-remarkable.

**Comment by ChiefORZ** [comment]
thanks for the fix! saved me a lot of time :)

File: README.md [hunk 1]
```diff
@@ -33,6 +34,7 @@ var MyComponent = React.createClass({
           <SomeOtherAmazingComponent />
```
Comment (chibicode) [inline README.md:34]: @matthewmueller I think you need to split the template string like the following in order to render `<SomeOtherAmazingComponent />` as a sub-component (that's what @acdlite's code's original intention was, I think):

``` jsx

{`
  ## Reasons React is great

  1. Server-side rendering
  2. This totally works:
`}

<SomeOtherAmazingComponent />

{`
  Pretty neat!
`}
```

**Comment by mxstbr** [comment]
Sorry, just now saw this patch but merged #23 first. 🙈  My bad, thanks for taking the time to contribute!

**Pull request closed without merging**